src/main/java/games/strategy/triplea/util/BonusIncomeUtils.java

```diff
@@ -8,26 +8,26 @@
 import games.strategy.util.IntegerMap;
 
 
-public class AiBonusIncomeUtils {
+public class BonusIncomeUtils {
 
   /**
    * Add AI bonus income based on the player's set percentage for all resources.
    * 
    * @return string that summarizes all the changes
    */
-  public static String addAiBonusIncome(final IntegerMap<Resource> income, final IDelegateBridge bridge,
+  public static String addBonusIncome(final IntegerMap<Resource> income, final IDelegateBridge bridge,
       final PlayerID player) {
     final StringBuilder sb = new StringBuilder();
     for (final Resource resource : income.keySet()) {
       final int amount = income.getInt(resource);
-      final int bonusPercent = Properties.getBonusIncomePercentage(player, bridge.getData());
-      final int bonusIncome = (int) Math.round(((double) amount * (double) bonusPercent / 100));
+      final int incomePercent = Properties.getBonusIncomePercentage(player, bridge.getData());
```

Comment:
Not sure about this, but should `Properties#getBonusIncomePercentage()` be renamed to `getIncomePercentage()`?

If so, might also consider renaming `Constants#getBonusIncomePercentageFor()`, `PlayerSelectorRow#bonusIncomePercentage`,  `PlayerSelectorRow#bonusIncomePercentageLabel`, and the `bonusLabel` local variable in `LocalSetupPanel#layoutComponents()`.